src/pytorch_lightning/strategies/ddp_spawn.py

```diff
@@ -71,6 +72,7 @@ def __init__(
         ddp_comm_wrapper: Optional[callable] = None,
         process_group_backend: Optional[str] = None,
         timeout: Optional[timedelta] = default_pg_timeout,
+        start_method: Literal["spawn", "fork", "forkserver"] = "spawn",
```

Comment:
Should forkserver be removed from the type:
```suggestion
        start_method: Literal["spawn", "fork"] = "spawn",
```
following https://github.com/Lightning-AI/lightning/pull/13405/files#r925506674?

Reply:
I don't understand. If someone wants to use it in a custom strategy, they can but would otherwise get a type error.